app/views/public/profile.html.erb

```diff
@@ -13,7 +13,7 @@
         <%= render partial: "public/mysavedpost"%>
       </div>
     </div>
-    <div class="col-4 p-2 ml-5" >
+    <div class="col-sm-4" >
       <%= render partial: "public/mydescription"%>
```

Comment:
use snake case for file names when there are more than one word

Reply:
Resolved by creating new Branch #113